src/components/application_manager/test/commands/hmi/simple_response_from_hmi_test.cc

```diff
@@ -318,6 +320,57 @@ TEST_F(OtherResponseFromHMICommandsTest, VIIsReadyResponse_Run_SUCCESS) {
   command->Run();
 }
 
+MATCHER_P(CheckMsgType, msg_type, "") {
+  return msg_type ==
+         static_cast<int32_t>(
+             (*arg)[am::strings::params][am::strings::message_type].asInt());
+}
+
+class NotificationFromHMITest
+    : public CommandsTest<CommandsTestMocks::kIsNice> {};
```

Comment:
@okozlovlux Just a question, not a remark. Why using an empty fixture is better than using no fixture?